### ISSUE TITLE
Learning rate increasing between epochs

### ISSUE THREAD

**Issue by PertuyF** [MouseLand/cellpose#96]
Hi!

First, thanks a lot for your work, Cellpose's results out of the box looks really promising, I love it.

I am trying to train a custom model to fit some special data of mine, and it looks like `CellposeModel` will not only change learning rate between each epoch, but that it will actually have it increasing? (see ref to code below)

https://github.com/MouseLand/cellpose/blob/fe3a0f9c18323faffd05fffe760dc29db22624f6/cellpose/models.py#L674-L679

Looks like LR is actually set to ETA, which is an increasing sequence set from 0 to user-defined LR.

https://github.com/MouseLand/cellpose/blob/fe3a0f9c18323faffd05fffe760dc29db22624f6/cellpose/models.py#L657

So if I understand well, the model starts training at LR 0, and increases to reach user-defined LR for last epoch. This sounds counter-intuitive, I would expect LR to be fixed between epochs or to *decrease* along training. Do I may misunderstand the code or the intention?

**Comment by marius10p** [comment]
Ah, we should have specified that in the manuscript and in the code. We slowly increase the learning rate in the first 10 epochs, then we decrease it over the last 100 epochs. We do this so the model stabilizes early on, as is typical for some deep learning frameworks. Jumping straight to the maximum learning rate often results in the model exploding.
